javaparser-core/src/main/java/com/github/javaparser/ast/body/ModifierSet.java

```diff
@@ -30,6 +33,21 @@
 public final class ModifierSet {
 
     /* Definitions of the bits in the modifiers field.  */
+    
+    private static Map<javax.lang.model.element.Modifier, Integer> ENUM_TO_INT = new HashMap<javax.lang.model.element.Modifier, Integer>();
```

Comment:
Reflect 8 seems not do contain a reference to default among the modifiers, even in version 8
http://docs.oracle.com/javase/8/docs/api/java/lang/reflect/Modifier.html